app/models/UserEnrolments.scala

```diff
@@ -37,15 +35,15 @@ case class UserEnrolmentStatus(service: String, state: Option[String], enrolment
 object UserEnrolmentStatus {
 
   val dateFormat: String = "yyyy-MM-dd HH:mm:ss.SSS"
```

Comment:
I'd consider making dateFormat and formatter private. They are used internally by UserEnrolmentStatus and shouldn't be used by other classes.